### DISCUSSION TITLE
hexo有没有全局统一配置站外图片403的地方啊？

### DISCUSSION THREAD

**Discussion by elarbor** [hexojs/hexo#5452] [Question / Answer] [answered]
我现在每个md文件都加`<meta name="referrer" content="no-referrer" />`。我想有没有统一只配一次的地方？
[目前有111个md文件都配了](https://github.com/orgs/hexojs/discussions/5451#discussioncomment-9009290)
![image](https://github.com/hexojs/hexo/assets/43592730/d506bfcb-636a-4860-9367-831ae3459825)

**Comment by stevenjoezhang** [comment, marked as answer]
在Hexo博客的`scripts`目录下创建一个js文件，名称随意（比如`no-referrer.js`），然后输入如下内容
```js
hexo.extend.injector.register('head_end', '<meta name="referrer" content="no-referrer" />', 'default');
```
之后重新`hexo clean` `hexo g`应该就行了
